changehc/delphi_changehc/run.py

```diff
@@ -14,11 +14,12 @@
 from delphi_utils import read_params
 
 # first party
-from .download_ftp_files import download
+from .download_ftp_files import download_covid, download_cli
+from .load_data import load_combined_data, load_cli_data
 from .update_sensor import CHCSensorUpdator
 
 
-def run_module():
+def run_module(): # pylint: disable=too-many-branches,too-many-statements
```

Comment:
Not keen on this disable -- is there a reason not to just split this up into subroutines? wouldn't that be easier to test anyhow?

Reply:
Done